maquetado/scss/_tech.scss

```diff
@@ -0,0 +1,40 @@
+.card {
+  background: $green;
+  border-radius: 5px;
+  height: 270px;
```

Comment:
Remember to use `max-height` property. Check this [style guide](https://github.com/Wolox/tech-guides/blob/master/frontend/docs/css-style-guide.md#setting-a-fixed-width)